unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/Debugging/Performance/PerformanceMeterController/PerformanceMeterController.cs

```diff
@@ -163,15 +173,17 @@ private void OnMetricsChange(PerformanceMetricsData newData, PerformanceMetricsD
                 secondsConsumed = Time.timeSinceLevelLoad - lastSavedSample.currentTime;
             }
 
-            SampleData newSample = new SampleData()
+            float frameTimeMs = Time.deltaTime * 1000f;
```

Comment:
We could use `ProfilerRecorder.StartNew(ProfilerCategory.Internal, "Main Thread")` to collect the samples for frame time.

Reply:
I'm not sure as well, but would improve consistency.